README.md

```diff
@@ -27,7 +27,11 @@ The market lacks a tool for developers to compare different database products be
   * [x] Data Refinement Menu
   * [x] Table for checked Instance
   * [ ] RAM / CPU wise calculator special for GCP
-* [ ] SEO with Nuxt (**On Going**) 
+* [ ] Cost Charts (**On Going**) 
+  * [ ] Compare the difference in monthly price between different offers ( Line Chart )
+  * [ ] Compare the difference in total price between different offers ( Accumulate Bar Chart )
```

Comment:
Not sure, what does accumulate bar chart mean. I thought it was just a simple bar char with instance spec as the x label?

Reply:
The bar has several dimension, like up-front commitment and on demand fee with different available rate.